test/turing.jl

```diff
@@ -1,28 +1,25 @@
 # Unconditioned coinflip model with `N` observations.
-@model function coinflip(; N::Int)
+@model function coinflip(y)
     p ~ Beta(1, 12)
-    y ~ filldist(Bernoulli(p), N)
+    y .~ Bernoulli(p)
```

Comment:
Would be worth adding a comment explaining why that change was made (to remove Turing from deps as the latest version of Turing was crashing as of March ~6)